CHANGELOG.md

```diff
@@ -42,11 +42,12 @@ The file browser now:
 
 ### Server-side execution and collaboration
 
-The previous release enabled extensions to implement server-side execution which preserves
-outputs cells created when the browser window with JupyterLab gets closed or disconnected.
+JupyterLab 4.2 release enabled extensions to implement server-side execution which preserves
+outputs created when the browser window with JupyterLab gets closed or disconnected.
```

Comment:
```suggestion
JupyterLab 4.2 added the option extensions to implement server-side execution, which preserves
outputs created when the JupyterLab browser window is closed or disconnected.
```

Reply:
> added the option extensions

Does not sound right?

Reply:
> JupyterLab browser window

Maybe we should drop JupyterLab altogether?

Reply:
```suggestion
JupyterLab 4.2 added the option for extensions to implement server-side execution, which preserves
outputs created when the browser window is closed or disconnected.
```